articles/virtual-machines/virtual-machines-instancemetadataservice-overview.md

```diff
@@ -17,93 +17,94 @@ ms.date: 03/27/2017
+By default, the Instance Metadata Service returns data in JSON format (`Content-Type: application/json`). However, different APIs can return data in different formats if requested.
+The following table is a reference of other data formats APIs may support.
 
-Element | Default Data Format | Other formats
+API | Default Data Format | Other Formats
 --------|---------------------|--------------
 /instance | json | text
 /scheduledevents | json | none
 
-For text format use format=text in the request URL, for example 
+To access a non-default response format, specify the requested format as a querystring parameter in the request. For example:
+
+    curl -H Metadata:true "http://169.254.169.254/metadata/instance?api-version=2017-03-01&format=text"
 
-```
-curl -H Metadata:true "http://169.254.169.254/metadata/instance?api-version=2017-03-01&format=text"
-```
 ### Security
-Instance metadata endpoint is accessible only from within the running virtual machine instance on a non-routable 169.254.169.254 IP address. In addition, any request with **X-Forwarded-For header** is rejected by the metadata service. 
-We also require requests to contain **Metadata:true** header being sent to ensure that the actual request was directly intended and not a part of unintentional redirection. 
+The Instance Metadata Service endpoint is accessible only from within the running virtual machine instance on a non-routable IP address. In addition, any request with a `X-Forwarded-For` header is rejected by the service.
+We also require requests to contain a `Metadata: true` header to ensure that the actual request was directly intended and not a part of unintentional redirection. 
+
 ### Error
-If there is a data element not found or malformed requests Instance metadata service returns standard HTTP Error, following are the few examples of return codes 
+If there is a data element not found or a malformed request, the Instance Metadata Service will return standard HTTP errors. For example:
 
-HTTP Return Code | Reason
+HTTP Status Code | Reason
 ----------------|-------
-200 | OK
-400 | Bad Request, missing header, pass -H Metadata:true
-404 | Not Found, requested element does’t exist 
-405 | Method not supported 
-429 | Too many requests, currently we only support maximum of 5 queries per second
+200 OK |
+400 Bad Request | Missing `Metadata: true` header
+404 Not Found | The requested element does't exist 
+405 Method Not Allowed | Only `GET` and `POST` requests are supported
+429 Too Many Requests | The API currently supports a maximum of 5 queries per second
 
 ### Examples
-#### Retrieving the network information 
+
+> [!NOTE] 
+> All API responses are JSON strings. All example responses below are pretty-printed for readability.
+
+#### Retrieving network information
 
 **Request**
-```
-curl -H Metadata:true "http://169.254.169.254/metadata/instance/network?api-version=2017-03-01"
 
-```
-**Response**
-> [!NOTE] 
```

Comment:
Why are you removing this, the json document returned as a string and doesnt appear formatted, the note was to make this clear